modules/platforms/cpp/tests/fake_server/connection_test.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "tests/client-test/ignite_runner_suite.h"
+#include "ignite/client/ignite_client.h"
+#include "fake_server.h"
+
+#include <gtest/gtest.h>
+#include <thread>
+
+using namespace ignite;
+
+class connection_test : public ignite_runner_suite {
+
+};
+
+
+TEST_F(connection_test, handshake_with_fake_server) {
```

Comment:
This is the `fake_server` package, no need to make the name longer.
```suggestion
TEST_F(connection_test, handshake_success) {
```